CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - iOS builds no longer break when native support disabled or not available ([#592](https://github.com/getsentry/sentry-unity/pull/592))
 
+### Features
+
+- Windows build-time debug symbol upload ([#607](https://github.com/getsentry/sentry-unity/pull/607))
```

Comment:
actually, I see we haven't had a generic windows-support changelog entry so maybe we should do that instead

Reply:
done